pkg/query/ui/ui.go

```diff
@@ -70,11 +82,18 @@ func (u *UI) status(w http.ResponseWriter, r *http.Request) {
 	u.executeTemplate(w, "status.html", struct {
 		Birth   time.Time
 		CWD     string
-		Version string
+		Version PrometheusVersion
```

Comment:
I think when building with promu (`make build` do that) we fill `-X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}` and things like that (see `.promu.yaml`)

Cannot we use that instead here?

Reply:
Well the root cause is that the template expects `Version` to be a struct and not a string: https://github.com/prometheus/prometheus/blob/master/web/ui/templates/status.html#L56

The info from `github.com/prometheus/common/version` is used to fill that struct, see line 90-95

Reply:
ah.. yea, but we are in control of this template, so we can change it as we want. But, sorry I see now what's the issue. 

All good, but let's just rename it to `ThanosVersion` maybe?

Reply:
Updated that